### DISCUSSION TITLE
Time to decimal time converter

### DISCUSSION THREAD

**Discussion by harishbewe** [iSoron/uhabits#1702] [Feature Requests]
If I am tracking  say, wheather my screen usage time is under control, and I am noting my daily screen usage time, but it is in 60 minutes format. But I guess the app takes mins as decimal values for 100. So it will be good to have in built time to decimal time converter.

**Comment by KraXen72** [comment]
this is a good idea. i track my sleeping hours (with a goal each night) and it's a bit cumbersome trying to calculate how much of an hour is 7hours and 23 minutes or something like that
